src/test/groovy/graphql/nadel/e2e/NadelE2ERenameHydrationTest.groovy

```diff
@@ -102,4 +107,103 @@ class NadelE2ERenameHydrationTest extends Specification {
         result.join().data == [updateSpecificIssue: [specificIssue: [name: [identity: "Luna"]]]]
     }
 
+    def 'hydration works when an ancestor field has been renamed'() {
```

Comment:
can you move this test into NadelExecutionStrategyTest2 and make suer you use the helper `test2Services` etc. ... thanks .

Reply:
Done, I didn't end up using a `test2Services` etc. because I needed to call the same service twice.

I could create a helper but didn't seem like it was worth the effort?